pkg/container/cri/yaml.go

```diff
@@ -28,17 +28,24 @@ Custom YAML (de)serialization for these types
 // UnmarshalYAML implements custom decoding YAML
 // https://godoc.org/gopkg.in/yaml.v3
 func (m *Mount) UnmarshalYAML(unmarshal func(interface{}) error) error {
-	type Alias Mount
-	aux := &struct {
-		Propagation string `yaml:"propagation"`
-		*Alias
-	}{
```

Comment:
sadly we have to give up this trick, yaml.v3 disallows duplicate keys.

Reply:
well, in the other said it forces us to be explicit :)

Reply:
yeah. this was a clever and concise trick though, I had fun discovering this :-)